src/docfx/SubCommands/DocumentBuilderWrapper.cs

```diff
@@ -156,7 +156,7 @@ private static IEnumerable<Assembly> LoadPluginAssemblies(string pluginDirectory
                 typeof(TocDocumentProcessor).Assembly,
                 typeof(SchemaDrivenDocumentProcessor).Assembly,
                 typeof(UniversalReferenceDocumentProcessor).Assembly,
-                typeof(MarkdigServiceProvider).Assembly
+                typeof(MarkdigServiceProvider).Assembly,
```

Comment:
>, [](start = 55, length = 1)

why add `,` here?

Reply:
following stylecop `SA1413UseTrailingCommasInMultiLineInitializers`

---
In reply to: [186641787](https://github.com/dotnet/docfx/pull/2732#discussion_r186641787) [](ancestors = 186641787)